modules/ROOT/pages/tgw-add-arm.adoc

```diff
@@ -0,0 +1,258 @@
+= Add Transit Gateway Attachments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
+
+
+Use Runtime Manager to add a transit gateway and connect your MuleSoft Virtual Private Cloud (VPC) to your private corporate network.
+
+
+== Prerequisites
+
+Before you can add a transit gateway:
+
+* Purchase an Anypoint VPC offering with Transit Gateway entitlements.
++
+The *Transit Gateways* page in Runtime Manager is visible to Anypoint organizations that have Transit Gateway entitlements. 
+* Sign in to an Anypoint Platform account with either the CloudHub Network Administrator or the CloudHub Network Viewer user permission.
++
+See xref:access-management::users.adoc#granting-permissions-and-roles-to-users[Granting Permissions and Roles to Users].
+* Set your default environment (in your profile) to an environment with Read Application permissions.
++
+See xref:access-management::managing-your-account.adoc#update-user-information[Update User Information].
+* Create an Anypoint VPC in Anypoint Platform.
++
+See xref:vpc-tutorial.adoc[Create and Update a VPC for Runtime Manager].
+* Create a transit gateway on your corporate AWS account.
++
+You must have access to this AWS account to add a transit gateway.
++
+See https://docs.aws.amazon.com/vpc/latest/tgw/tgw-getting-started.html[Getting started with transit gateways^] in the AWS documentation for information about creating a transit gateway on AWS.
++
+One transit gateway can support up to 10 VPC connections.
+* Identify the subnets in your network (in CIDR notation) that you want to make accessible through the transit gateway.
+
+
+== Overview 
+
+The steps for adding a Transit Gateway to your VPC are:
+
+. <<add_tgw,Add the transit gateway>> in Anypoint Platform.
+. <<create_resource_share,Create a Resource Share in AWS>> to share your transit gateway with Anypoint Platform.
+. <<attach_vpc_tgw,Attach your VPCs>> to the transit gateway.
+. <<accept_attachment,Accept the VPC attachments>> in AWS.
```

Comment:
They attach their VPCs to the TGW by…

Id

1. entifying a VPC to attach (MuleSoft then automatically creates the attachment)
2. Accepting the attachment MuleSoft creates
3. Repeating as necessary for other VPCs

I think you can still say "Attach your VPCs to the transit gateway," but in that case I wouldn't list "Accept the VPC attachments" as a separate step.

Reply:
@taylordolan - I separated them out because they're two different processes (Anypoint side and AWS side)/sections.

I changed it to combine them into a single step in this list.